api_tests/harness.js

```diff
@@ -203,29 +209,40 @@ describe("Starting services", async function() {
             Object.keys(config.ledger_query_service).forEach(async function(
                 name
             ) {
-                await startLedgerQueryService(
+                const subprocess = await startLedgerQueryService(
                     name,
                     config.ledger_query_service[name]
                 );
+                subprocesses.push(subprocess);
             });
         }
 
         if (config.comit_node) {
             logger.info("++ Starting COMIT node(s)");
             Object.keys(config.comit_node).forEach(async function(name) {
-                await startComitNode(name, config.comit_node[name]);
+                const subprocess = await startComitNode(
+                    name,
+                    config.comit_node[name]
+                );
+                subprocesses.push(subprocess);
             });
         }
 
         await test_lib.sleep(2000);
     });
 
-    describe("Running Test Suite", async function() {
+    beforeEach(async function() {
+        // FIXME: use setInterval or equivalent.
+        // Note: could not make it work, Mochas was never stopping
```

Comment:
change this comment to: "Using `setInterval` to create a block every X seconds made the test harness hang (even after stopping the interval) so we just create a block before every test for now.